SSMs/SafeWriting/ServerFolder/server.cfg

```diff
@@ -0,0 +1,24 @@
+sv_servername = "SSM SafeWriting Server"
+sv_gamerules = PowerStruggle
+sv_cheatprotection = 0
+sv_maxplayers = 32
+
+sv_port = 21330
+net_pb_sv_enable false
+g_timelimit = 0
+g_minteamlimit = 1
+g_revivetime = 3
+g_autoteambalance = 1
+g_teamlock = 1
+g_friendlyfireratio = 0
+g_tk_punish = 0
+g_tk_punish_limit = 10
+
+log_verbosity = 0
+log_fileverbosity = 0
+ban_timeout = 1440
+
+sv_master m.crymp.net
+
```

Comment:
`GSMasterHook` is used only by client exe and it uses `cl_master` cvar. I think server shouldn't need it at all.

```suggestion
```